package.json

```diff
@@ -131,7 +135,7 @@
   "dependencies": {
     "@inrupt/solid-client": "^2.0.0",
     "@inrupt/solid-client-errors": "^0.0.2",
-    "@inrupt/solid-client-vc": "^1.1.2",
+    "@inrupt/solid-client-vc": "^1.2.0",
```

Comment:
This adds query endpoint discovery, but its usage is currently blocked by a small ESS bug.